### ISSUE TITLE
FM updated docs

### ISSUE THREAD

**Issue by Auriostefm** [ONRR/onrr.gov-site#427]
Debt Collection Assignment doc updated to be 508 compliant.  I cannot attach the document.

Original location:
https://www.onrr.gov/ReportPay/PDFDocs/DebtCollectionAssign.pdf

Thank you 
Antoinette Urioste
303-231-3142

**Comment by Maroyafaied** [comment]
Hi Antoinette, 

You can drag the file to the comment section and it will attach it. If that doesn't work, feel free to email it to onrrweb@onrr.gov

**Comment by Auriostefm** [comment]
Thank you.  I will be emailing all the documents.

Respectfully,
Antoinette



Antoinette Urioste
Financial Management
Revenue, Reporting & Compliance Management
Office of Natural Resources Revenue
antoinette.urioste@onrr.gov<mailto:antoinette.urioste@onrr.gov>
P: (303) 231-3142

**Comment by Maroyafaied** [comment]
[DebtCollectionAssign.pdf](https://github.com/ONRR/onrr.gov-site/files/5777288/DebtCollectionAssign.pdf)
[ReporterLetter_Solid Minerals_june 2019 508.pdf](https://github.com/ONRR/onrr.gov-site/files/5777290/ReporterLetter_Solid.Minerals_june.2019.508.pdf)
[5 fedwire.pdf](https://github.com/ONRR/onrr.gov-site/files/5777291/5.fedwire.pdf)

@Auriostefm , I've attached all 3 docs in this issue. I will close out the other two issues and assign one person to check for accessibility errors and upload all the 3 docs . We will notify you on here if there are any accessibility errors, if not we will ask you to review the development site before we publish the updates on the live site. Thanks!

**Comment by Auriostefm** [comment]
Thank you!

Respectfully,
Antoinette



Antoinette Urioste
Financial Management
Revenue, Reporting & Compliance Management
Office of Natural Resources Revenue
antoinette.urioste@onrr.gov<mailto:antoinette.urioste@onrr.gov>
P: (303) 231-3142

**Comment by lpgoldstein** [comment]
@Auriostefm There is an issue with the header in the debt collection pdf. Since it's a one page table, the easiest thing to do would be to turn it into an excel sheet. I have attached the accessibility report and added a link that will help you fix the header issue if you choose to keep the table as a pdf. https://helpx.adobe.com/acrobat/using/editing-document-structure-content-tags.html#add_alternate_text_and_supplementary_information_to_tags
[debt collection assignments accessibility report .docx](https://github.com/ONRR/onrr.gov-site/files/5777461/debt.collection.assignments.accessibility.report.docx)

**Comment by Auriostefm** [comment]
Hello,
I don't understand the header issue.  Wasn't the header tagged as a header?

Respectfully,
Antoinette



Antoinette Urioste
Financial Management
Revenue, Reporting & Compliance Management
Office of Natural Resources Revenue
antoinette.urioste@onrr.gov<mailto:antoinette.urioste@onrr.gov>
P: (303) 231-3142

**Comment by lpgoldstein** [comment]
@Auriostefm The accessibility report says that it isn't tagged as a header so you probably just have to go in and redo the tag.

**Comment by lpgoldstein** [comment]
@Auriostefm  The new dear reporter letter is ready for your review: http://2017onrr/DearRep.htm as is the fedwire document http://2017onrr/ReportPay/payments.htm#FP

**Comment by Auriostefm** [comment]
I went in and tagged multiple times, for some reason, it still doesn't like it.  I think because the header is the header of the document and the table.

Respectfully,
Antoinette



Antoinette Urioste
Financial Management
Revenue, Reporting & Compliance Management
Office of Natural Resources Revenue
antoinette.urioste@onrr.gov<mailto:antoinette.urioste@onrr.gov>
P: (303) 231-3142

**Comment by lpgoldstein** [comment]
@Auriostefm what about differentiating between the two?

**Comment by lpgoldstein** [comment]
@Auriostefm I'll take another look.

**Comment by Auriostefm** [comment]
I have exported to a spreadsheet.  How do I go about checking for accessibility?

Respectfully,
Antoinette



Antoinette Urioste
Financial Management
Revenue, Reporting & Compliance Management
Office of Natural Resources Revenue
antoinette.urioste@onrr.gov<mailto:antoinette.urioste@onrr.gov>
P: (303) 231-3142

**Comment by lpgoldstein** [comment]
Confirmed all via email.
